File: LV0/마지막 두 원소.js
function solution(num_list) {
  const length = num_list.length;
  const last = num_list[length - 1];
  const beforeLast = num_list[length - 2];
  if (last > beforeLast) {
    num_list.push(last - beforeLast);
  } else {
    num_list.push(last * 2);
  }

  return num_list;
}

File: LV0/마지막 두 원소.py
def solution(num_list):
    length = len(num_list)
    last = num_list[length - 1]
    beforeLast = num_list[length - 2]
    if last > beforeLast:
        num_list.append(last - beforeLast)
    else:
        num_list.append(last * 2)

    return num_list